src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -457,91 +457,24 @@
 .clickable-style--link {
   @mixin textLink;
 
-  padding: 0;
+  border-radius: 0;
   border: 0;
-  text-align: left;
   background-color: transparent;
-  gap: 0.25em; /* 2 */
-}
-
-/**
- * Link brand clickable style
- */
-.clickable-style--link.clickable-style--brand {
-  color: var(--eds-theme-color-button-link-brand);
-
-  &:hover {
-    color: var(--eds-theme-color-button-link-brand-hover);
-  }
-
-  &:active {
-    color: var(--eds-theme-color-button-link-brand-active);
-  }
 }
 
 /**
  * Link neutral clickable style
  */
 .clickable-style--link.clickable-style--neutral {
-  color: var(--eds-theme-color-button-link-neutral);
-
-  &:hover {
-    color: var(--eds-theme-color-button-link-neutral-hover);
-  }
-
-  &:active {
-    color: var(--eds-theme-color-button-link-neutral-active);
-  }
-}
-
-/**
- * Link success clickable style
- */
-.clickable-style--link.clickable-style--success {
-  color: var(--eds-theme-color-button-link-success);
+  text-decoration-color: var(--eds-theme-color-icon-neutral-strong);
 
   &:hover {
-    color: var(--eds-theme-color-button-link-success-hover);
-  }
-
-  &:active {
-    color: var(--eds-theme-color-button-link-success-active);
-  }
-}
-
-/**
- * Link warning clickable style
- */
-.clickable-style--link.clickable-style--warning {
-  color: var(--eds-theme-color-button-link-warning);
-
-  &:hover {
-    color: var(--eds-theme-color-button-link-warning-hover);
-  }
-
-  &:active {
-    color: var(--eds-theme-color-button-link-warning-active);
-  }
-}
-
-/**
- * Link error clickable style
- */
-.clickable-style--link.clickable-style--error {
-  color: var(--eds-theme-color-button-link-error);
-
-  &:hover {
-    color: var(--eds-theme-color-button-link-error-hover);
-  }
-
-  &:active {
-    color: var(--eds-theme-color-button-link-error-active);
+    text-decoration-color: var(--eds-theme-color-border-neutral-strong);
   }
 }
 
-/* Style icons that may be in the clickable style children */
-.clickable-style--link svg {
-  --icon-size-default: 1.25em;
+.clickable-style--link.clickable-style--neutral::before {
```

Comment:
Is this supposed to append to / override the `::before` pseudoelement in the `textLInk` mixin? Can you add a comment for this block?

Reply:
Yeah it overrides it for situations where the purple bg might not mesh well.

Reply:
Added a comment, thanks for the suggestion!